src/http-gateways/path-gateway.md

```diff
@@ -567,7 +591,10 @@ The following response types require an explicit opt-in, can only be requested w
 - Raw Block (`?format=raw`)
   - Opaque bytes, see [application/vnd.ipld.raw](https://www.iana.org/assignments/media-types/application/vnd.ipld.raw).
 - CAR (`?format=car`)
```

Comment:
... but that currently such an error case must be determined by the consumer of the CAR in lieu of a good [error signalling mechanism](https://github.com/ipfs/specs/pull/332).

Reply:
In this particular case the consumer of the CAR should be able to determine this as well so that they can't be lied to. For example, if you ask for `/ipfs/bafyDir/file` and it exists but I return to you an error saying it doesn't you should know I'm wrong and be able to ask someone else for the data.

Reply:
IMO it is out of scope: malicious server will not send proof anyway, just HTTP 404 without body, so a robust client will have to have recovery logic anyway for manually traversing the path segment-by-segment, using multiple gateways, until a proof of non-existence is obtained.